test/schema/cgmSettingsTest.js

```diff
@@ -0,0 +1,141 @@
+    helper.expectStringField(goodObject, 'transmitterId');
+  });
+  describe('units', function () {
+    helper.rejectIfAbsent(goodObject, 'units');
+    helper.expectStringField(goodObject, 'units');
+    helper.expectFieldIn(goodObject, 'units',
+      ['mmol/L', 'mmol/l', 'mg/dL', 'mg/dl'],
+      ['mmol/L', 'mmol/L', 'mg/dL', 'mg/dL']);
+  });
+  describe('highAlerts', function () {
+    helper.rejectIfAbsent(goodObject, 'highAlerts');
+    helper.expectObjectField(goodObject, 'highAlerts');
+
+    it('fields', function(done){
+      var localGood = _.cloneDeep(goodObject);
+      helper.rejectIfAbsent(localGood.lowAlerts, 'enabled');
+      helper.expectObjectField(localGood.lowAlerts, 'enabled');
+      helper.rejectIfAbsent(localGood.lowAlerts, 'level');
+      helper.expectNumericalField(localGood.lowAlerts, 'level');
+      helper.rejectIfAbsent(localGood.lowAlerts, 'snooze');
+      helper.expectNumericalField(localGood.lowAlerts, 'snooze');
+      done();
+    });
+  });
+  describe('lowAlerts', function () {
+    helper.rejectIfAbsent(goodObject, 'lowAlerts');
+    helper.expectObjectField(goodObject, 'lowAlerts');
+
+    it('fields', function(done){
+      var localGood = _.cloneDeep(goodObject);
+      helper.rejectIfAbsent(localGood.lowAlerts, 'enabled');
+      helper.expectObjectField(localGood.lowAlerts, 'enabled');
+      helper.rejectIfAbsent(localGood.lowAlerts, 'level');
+      helper.expectNumericalField(localGood.lowAlerts, 'level');
+      helper.rejectIfAbsent(localGood.lowAlerts, 'snooze');
+      helper.expectNumericalField(localGood.lowAlerts, 'snooze');
+      done();
+    });
+  });
+  describe('outOfRangeAlerts', function () {
+    helper.expectObjectField(goodObject, 'outOfRangeAlerts');
+    it('fields', function(done){
+      var localGood = _.cloneDeep(goodObject);
+      helper.rejectIfAbsent(localGood.outOfRangeAlerts, 'enabled');
+      helper.expectObjectField(localGood.outOfRangeAlerts, 'enabled');
+      helper.rejectIfAbsent(localGood.outOfRangeAlerts, 'snooze');
+      helper.expectNumericalField(localGood.outOfRangeAlerts, 'snooze');
+      done();
+    });
+  });
+  describe('rateOfChangeAlerts', function () {
+    helper.rejectIfAbsent(goodObject, 'rateOfChangeAlerts');
+    helper.expectObjectField(goodObject, 'rateOfChangeAlerts');
+    it('fallRate', function(done){
+      var localGood = _.cloneDeep(goodObject);
+      helper.rejectIfAbsent(localGood, 'fallRate');
+      helper.expectObjectField(localGood, 'fallRate');
+      helper.rejectIfAbsent(localGood.fallRate, 'enabled');
+      helper.expectObjectField(localGood.fallRate, 'enabled');
+      helper.rejectIfAbsent(localGood.fallRate, 'rate');
+      helper.expectNumericalField(localGood.fallRate, 'rate');
+      done();
+    });
+    it('riseRate', function(done){
+      var localGood = _.cloneDeep(goodObject);
+      helper.rejectIfAbsent(localGood, 'riseRate');
+      helper.expectObjectField(localGood, 'riseRate');
```

Comment:
Shouldn't this be `expectNumericalField` and why does it pass with `expectObjectField`?